dozer-api/src/grpc/client_server.rs

```diff
@@ -56,13 +56,18 @@ impl ApiServer {
             builder = builder.register_encoded_file_descriptor_set(descriptor_bytes);
         }
         let inflection_service = builder.build().map_err(GrpcError::ServerReflectionError)?;
+        let dozer_master_secret = std::env::var("DOZER_MASTER_SECRET").ok();
+        let security = self
+            .security
+            .clone()
+            .or_else(|| dozer_master_secret.map(ApiSecurity::Jwt));
```

Comment:
Learnt from @karolisg, environment variable should override configuration.

Reply:
hmm, i think should not in this case, we should prior the secret user input in `dozer-config.yaml` right, since they have no idea about this variable. We only take this value if user does not specify secret in api security config

Reply:
From dozer core's point of view, env var and config are both user input.

It's the cloud orchestrator manipulating user input, and it should (and it already does) ensure that it respects value in config.

Reply:
sure, let me update with that rule: variable should override configuration